snapshots/sample/app/build.gradle.kts

```diff
@@ -28,11 +28,13 @@ android {
     }
     debug {
       applicationIdSuffix = ".debug"
-      sourceSets {
-        getByName("androidTest") {
-          java.srcDir("generated/ksp/debugAndroidTest/kotlin")
```

Comment:
This is no longer necessary as of KSP 1.0.9: https://kotlinlang.org/docs/ksp-quickstart.html#make-ide-aware-of-generated-code